kernel-cert-script

```diff
@@ -0,0 +1,86 @@
+#!/bin/sh
+
+op=$(echo "$0" | sed -e "s/.*-//")
+
+while true ; do
+    case $1 in
+	--kmp)
+	    kmp="$2"
+	    shift 2
+	    ;;
+	--certs)
+	    certs="$2"
+	    shift 2
+	    ;;
+	*) break
+	    ;;
+    esac
+done
+
+run_mokutil () {
+    [ -z "$KERNEL_PACKAGE_SCRIPT_DEBUG" ] || echo mokutil "$@" > /dev/kmsg
+    mokutil "$@"
```

Comment:
The messages that go to /dev/stderr tend to scroll by and get lost unless the user takes measures in advance. the terminal scrollback is typically not long enough for a zypper run.

But if you are setting a variable you can as well set up logging of your zypper output so no strong opinion on this one.

Reply:
Note that depending on logging settings, **journald** may suppress log messages from `kmsg` that don't originate from the kernel. `>&2` is almost always the right thing to do, it makes the messages end up where the calling process sends them. This could be a log file, the journal, a terminal, or perhaps `/dev/kmsg` or `/dev/null`.

Reply:
I'm still not happy with this. My suggestion was `>&2`, not `2>&1`. We should keep stdout free of log messages and error messagse.